src/redux/books/books.js

```diff
@@ -1,40 +1,34 @@
-export const booksState = [{
-  id: '1',
-  category: 'Adventure',
-  title: 'Warriors',
-  author: 'Erin Hunter',
-},
-{
-  id: '2',
-  category: 'Action',
-  title: 'The Good Soldier',
-  author: 'L. T. Ryan',
-},
-{
-  id: '3',
-  category: 'Self Help',
-  title: 'Cracking the Coding Interview',
-  author: 'Gayle Laakmann McDowell',
-}];
+import BookStoreUtility from '../../BookstoreAPI/apiUtility';
 
+export const booksState = [];
+
+const GET_BOOKS_SUCCESS = 'bookStore/books/GET_BOOKS_SUCCESS';
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
 
-export function addBook(book) {
-  return { type: ADD_BOOK, book };
+export function getBooks(books) {
+  return { type: GET_BOOKS_SUCCESS, books };
 }
 
-export function removeBook(id) {
-  return { type: REMOVE_BOOK, id };
-}
+export const addBook = (book) => async (dispatch) => {
+  await BookStoreUtility.apiToAddBook(book);
+  dispatch({ type: ADD_BOOK, book });
+};
+
+export const removeBook = (id) => async (dispatch) => {
+  await BookStoreUtility.apiToRemoveBook();
+  dispatch({ type: REMOVE_BOOK, id });
+};
```

Comment:
- [ ] **Required:** You have done great work on this milestone. Well done 👏👏

However, I have observed that your remove doesn't remove books from the API. Kindly fix this behaviour.

![Remove_from_api](https://user-images.githubusercontent.com/27709832/167254678-4a4ae2f7-e78d-497b-8cda-1f031cf87d2f.gif)